src/components/VSelect/VSelectList.js

```diff
@@ -144,23 +144,16 @@ export default {
         tile.componentOptions.Ctor.options.name !== 'v-list-tile'
     },
     getAvatar (item) {
-      return getObjectValueByPath(item, this.itemAvatar)
+      return Boolean(getPropertyFromItem(item, this.itemAvatar, false))
     },
     getDisabled (item) {
-      return getObjectValueByPath(item, this.itemDisabled)
+      return Boolean(getPropertyFromItem(item, this.itemDisabled, false))
     },
     getText (item) {
-      return this.getPropertyFromItem(item, this.itemText)
+      return (getPropertyFromItem(item, this.itemText, item) || '').toString();
```

Comment:
Maybe `String()...`? depends how we want to handle falsy values I guess. 

Remove the semicolon too please.